data-otservbr-global/scripts/bestiary/charms.lua

```diff
@@ -198,7 +198,7 @@ local charms = {
 		name = "Vampiric Embrace",
 		description = "Adds 4% Life Leech to attacks if wearing equipment that provides life leech.",
 		type = CHARM_PASSIVE,
-		percent = 4,
+		percent = 400,
```

Comment:
Is this right? Shouldn't it stay at 4%?

Reply:
I read the code below, the key on the table is a bit unfortunate now since this is a 0.01% increment, but yeah 400 is 4% huh